### ISSUE TITLE
 EXC_BAD_ACCESS on Release build 

### ISSUE THREAD

**Issue by AAAstorga** [quickbirdstudios/XCoordinator#144]
Hello, 
I am running into a crash on release builds.

Trying to build with Xcode 11.2.1, target iOS 13 and above.

I tested with my personal app and the Xcoordinator Example app and both are the same result.

I wasn't seeing this before in Release builds. It will build successfully in Debug builds, but once I change the scheme to Release, it crashes on EXC_BAD_ACCESS.

Let me know if you can't repro and need more information.

<img width="825" alt="image" src="https://user-images.githubusercontent.com/15990084/68409046-4923c280-013b-11ea-8f3d-fe9f2bd20e59.png">
(Screenshot from Xcoordinator Example app, but I also had the same result on my personal app)

**Comment by pauljohanneskraft** [comment]
Hey - thanks for the notification! 

We recently also encountered the same problem in some of our own apps. I could also reproduce the problem in the XCoordinator-Example app. However, when I switched the dependency management of the XCoordinator-Example app over to Cocoapods instead of Swift Package Manager, it did no longer crash - That means, it is probably a configuration problem of the framework, the client or a Xcode/Swift Package Manager bug. We will let you know once we found a solution to the problem.
